hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/MemDefinitionsServiceImpl.java

```diff
@@ -41,6 +41,7 @@
 import org.hawkular.alerts.api.model.condition.ThresholdRangeCondition;
 import org.hawkular.alerts.api.model.dampening.Dampening;
 import org.hawkular.alerts.api.model.trigger.Trigger;
+import org.hawkular.alerts.api.model.trigger.Trigger.Mode;
```

Comment:
Side comment: I left here the MemDefinitionsServiceImpl.java just to have an "example" of a different implementation. But I am ok if we dropped as DbDefinitionsServiceImpl is the important one.
Both are in "draft/demo/poc" status, so perhaps just one can be enough and we don't need to maintain it.